synapse/logging/context.py

```diff
@@ -221,14 +220,13 @@ class ContextRequest:
 class _Sentinel:
     """Sentinel to represent the root context"""
 
-    __slots__ = ["previous_context", "finished", "request", "scope", "tag"]
+    __slots__ = ["previous_context", "finished", "request", "tag"]
```

Comment:
We don't need the `opentracing` `scope` stuff in here anymore.

The `scopecontextmanager.py` is no more. Doesn't seem necessary to make the context work across logging contexts. Twisted now better supports `ContextVar` stuff better (see https://github.com/matrix-org/synapse/pull/13400#discussion_r936195492)